app/oxalis/ndstore/NDChannel.scala

```diff
@@ -3,21 +3,23 @@
  */
 package oxalis.ndstore
 
-import com.scalableminds.braingames.binary.models.DataLayer
+import com.scalableminds.braingames.binary.models.datasource.{Category, ElementClass}
 import play.api.data.validation.ValidationError
 import play.api.libs.functional.syntax._
 import play.api.libs.json.{JsResult, JsValue, _}
 
-case class NDChannel(name: String, dataType: String, channelType: String)
+case class NDChannel(name: String, dataType: ElementClass.Value, channelType: Category.Value)
 
 object NDChannels {
 
   object NDChannelsReads extends Reads[List[NDChannel]] {
     val channelBodyReads =
-      ((__ \ 'datatype).read[String].filter(
-        ValidationError("ndstore.invalid.data.type"))(DataLayer.isValidElementClass) and
-        (__ \ 'channel_type).read[String].filter(
-          ValidationError("ndstore.invalid.channel.type"))(ND2WK.channelTypeMapping.contains)).tupled
+      // TODO jfrohnhofen check this verification works and move channelTypeMapping to enum???
```

Comment:
done. the validation works (tested ndstore with daniel) and it doesn't really make sense to use the `Category` enum here, since `channelType` uses different string constants